src/test/java/com/github/fishio/gui/TestAchievementScreen.java

```diff
@@ -0,0 +1,31 @@
+package com.github.fishio.gui;
```

Comment:
There is a mistake in switchToScreen, the screen is called "achievementScreen", not "AchievementScene".
Also, pull request #64 makes some GuiTest changes that makes GuiTests more stable, which might prevent other problems.